src/HttpCommon.jl

```diff
@@ -3,7 +3,6 @@ __precompile__()
 module HttpCommon
 
 using Compat
-using Compat: UTF8String
```

Comment:
Indeed not - removed!
I imagine `Compat` will return, though, when `type` is deprecated (in 0.7/1.0?) and replaced with `struct` (possibly `mutable`).